docs/tables/crowdstrike_detection.md

```diff
@@ -0,0 +1,39 @@
+# Table: crowdstrike_detection
+
+Detections are events identified by Falcon sensors on the hosts in your environment.
+
+## Examples
```

Comment:
Can you please add more 1-2 examples here? Detections seem like they have a lot of fields, so I'm sure there are interesting ways to pull information from detections, or pull specific detections based on behaviours, confidence, severity, etc.